sklearn/covariance/tests/test_graph_lasso.py

```diff
@@ -69,9 +127,9 @@ def test_graph_lasso_cv(random_state=1):
     try:
         sys.stdout = StringIO()
         # We need verbose very high so that Parallel prints on stdout
-        GraphLassoCV(verbose=100, alphas=3, tol=1e-1).fit(X)
+        GraphLassoCV(verbose=100, alphas=5, tol=tol).fit(X)
```

Comment:
My preference would be to add a separate tolerance for the enet, but I was a bit hesitant since I'm new to the project. If you guys agree that it makes sense to separate them then I will make the change; I can certainly imagine wanting to change the graph_lasso tolerance but don't see why anyone would ever want to mess with the enet precision.